frontend/pages/club/[club]/edit.js

```diff
@@ -312,13 +312,28 @@ class ClubForm extends Component {
   }
 
   render() {
-    const { authenticated, schools, majors, years, tags } = this.props
+    const { authenticated, userInfo, schools, majors, years, tags } = this.props
     const { club, invites, editMember } = this.state
 
     if (authenticated === false) {
       return <AuthPrompt />
     }
 
+    if (
+      authenticated &&
+      club &&
+      !userInfo.is_superuser &&
+      !userInfo.membership_set.some(m => m.code === club.code && m.role <= 10)
+    ) {
+      return (
+        <AuthPrompt title="Oh no!" hasLogin={false}>
```

Comment:
Unless AuthPrompt has prop-types enforcing hasLogin be a boolean, you might be able to just delete hasLogin - when AuthPrompt tries to pull hasLogin, it will be undefined, which is falsey.

Reply:
Ah I set the default value of `hasLogin` to true, which displays the login button.

The login button does not apply in the case of access denied, since the user is logged in but does not have the right permissions.